File: 프로그래머스/0/120834. 외계행성의 나이/외계행성의 나이.java
class Solution {
    public String solution(int age) {
        String answer = "";
        char[] alpha = {'a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j'};
        String ageStr = String.valueOf(age);
        
        for (int i = 0; i < ageStr.length(); i++) {
            int digit = Character.getNumericValue(ageStr.charAt(i));
            answer += alpha[digit];
        }
        return answer;
    }
}

File: 프로그래머스/0/120834. 외계행성의 나이/외계행성의 나이.py
def solution(age):
    answer = ''
    info = {0 : 'a', 1 : 'b', 2 : 'c', 3:'d', 4:'e', 5:'f', 6:'g', 7:'h', 8:'i', 9:'j'}
    tmp = ''
    while age > 0:
        n = age % 10
        tmp += info[n]
        age //= 10
    for i in range(len(tmp)):
        answer += tmp[len(tmp)-1-i]
    return answer